src/matchers/mock/toBeRequestedWithResponse.ts

```diff
@@ -0,0 +1,40 @@
+import { waitUntil, enhanceError } from '../../utils'
+import { runExpect } from '../../util/expectAdapter'
+import { equals } from '../../jasmineUtils'
```

Comment:
I'd use the framework's builtin matchers whether it's Jest or Jasmine but have no idea how to do it better atm.

I believe it could be a good idea to wrap builtin matcher first to avoid possible conflicts as we discussed before https://github.com/webdriverio/expect-webdriverio/issues/113. 
Once it's done let's see if we can remove jasmineUtils file.

Reply:
@mgrybyk are you suggesting that the matcher should look like this then?

```js
const pass = await waitUntil(async () => {
    expect(mock?.body).toEqual(response)
})
```

Given that this requires a bit more refactor (#113) how about moving this into a separate issue and use this `jasmineUtils` file for now?

Reply:
let's keep using jasmine utils for now